src/target/riscv/riscv.c

```diff
@@ -5031,6 +5005,56 @@ COMMAND_HANDLER(riscv_set_enable_trigger_feature)
 	return ERROR_OK;
 }
 
+static COMMAND_HELPER(report_reserved_triggers, struct target *target)
+{
+	RISCV_INFO(r);
+	if (riscv_enumerate_triggers(target) != ERROR_OK)
+		return ERROR_FAIL;
+	const char *separator = "";
+	for (riscv_reg_t t = 0; t < r->trigger_count; ++t)
+		if (r->reserved_triggers[t]) {
+			command_print_sameline(CMD, "%s%" PRIu64, separator, t);
+			separator = " ";
+		}
```

Comment:
Detail: If checkpatch allows it, I'd wrap the for-loop body into another pair of curly braces.

```suggestion
	for (riscv_reg_t t = 0; t < r->trigger_count; ++t) {
		if (r->reserved_triggers[t]) {
			command_print_sameline(CMD, "%s%" PRIu64, separator, t);
			separator = " ";
		}
	}
```

Reply:
Addressed.